contracts/core/Dispatcher.sol

```diff
@@ -379,7 +417,7 @@ contract Dispatcher is OwnableUpgradeable, UUPSUpgradeable, IDispatcher {
 
         // prove absence of packet receipt on Polymer chain
         // TODO: add non membership support
-        _lightClient.verifyNonMembership(proof, "packet/receipt/path");
+        lightClient.verifyNonMembership(proof, "packet/receipt/path");
```

Comment:
I changed to use Ibc.packetcommitment proof key, but FYI we still have yet to implement the actual `lightClient.verifyNonMembershipProof` method

Reply:
so current behavior is we don't support timeout packets for op-client since it'd throw. 
do you think it's worth a test showing timeout packets will revert for op-client ?